app/views/home/index.html.erb

```diff
@@ -73,7 +73,7 @@
         <div class="col-lg-4">
           <h4>About</h4>
           <div class="hline-w"></div>
-          <p>AskUp is a joint venture between investigators at Beth Israel Deaconess Medical Center (BIDMC) and engineers in the Teaching and Learning Technologies (TLT) program at Harvard University. </p>
+          <p>AskUp is a joint venture between investigators at Beth Israel Deaconess Medical Center (BIDMC) and engineers in the Teaching and Learning Technologies (TLT) program at Harvard University. Our project was awarded a Spark Grant from the Harvard Initiative for Teaching & Learning (HILT)</p>
```

Comment:
Please change the ampersand to `&amp;` as per the recommendations here: 
http://www.w3schools.com/html/html_entities.asp